cli/local/docker_spec.go

```diff
@@ -88,9 +88,9 @@ func getAPIEnv(api *spec.API, awsClient *aws.Client) []string {
 		"CORTEX_MODELS="+strings.Join(api.ModelNames(), ","),
 		"CORTEX_API_SPEC="+filepath.Join("/mnt/workspace", filepath.Base(api.Key)),
 		"CORTEX_PROJECT_DIR="+_projectDir,
-		"CORTEX_WORKERS_PER_REPLICA=1",
-		"CORTEX_THREADS_PER_WORKER=1",
-		"CORTEX_MAX_WORKER_CONCURRENCY=1000",
+		"CORTEX_PROCESSES_PER_REPLICA="+s.Int32(api.Predictor.ProcessesPerReplica),
+		"CORTEX_THREADS_PER_PROCESS="+s.Int32(api.Predictor.ThreadsPerProcess),
+		"CORTEX_MAX_PROCESS_CONCURRENCY=1000",
```

Comment:
maybe this should match what we do [here](https://github.com/cortexlabs/cortex/blob/refactor/parallelism-fields/pkg/operator/operator/k8s_specs.go#L634), but replace `api.Autoscaling.MaxReplicaConcurrency` with `1024` (since that is the default value and you can't update it for local). Or maybe `api.Autoscaling.MaxReplicaConcurrency` should be moved out of autoscaling too (that could be a separate PR, and less urgent). What do you think?

Reply:
Yes, you are totally right.

Reply:
I'm also thinking that if `MaxReplicaConcurrency` is taken out, then so `TargetReplicaConcurrency` should be. Probably move all these 4 fields in a separate section too called `parallelism`, or `concurrency`.

Reply:
Yes that could make sense. We can discuss this more in the future, for now I think we can go ahead with this PR.